src/d3a/models/market/market_structures.py

```diff
@@ -408,3 +409,17 @@ def parse_event_and_parameters_from_json_string(payload) -> Tuple:
         kwargs["bid_trade"] = trade_from_json_string(kwargs["bid_trade"])
     event_type = MarketEvent(data["event_type"])
     return event_type, kwargs
+
+
+class AvailableMarketTypes(Enum):
```

Comment:
I have a similar enum in feature/D3ASIM-3589, but don't worry about it. I'll delete one of the 2 once both PRs are merged